charts/nutanix-csi-storage/values.yaml

```diff
@@ -132,30 +145,59 @@ servicemonitor:
 
 controller:
   replicas: 2
-  image: quay.io/karbon/ntnx-csi:v2.6.6
+  image: quay.io/karbon/ntnx-csi:v3.0.0-rc6
   nodeSelector: {}
   tolerations: []
+  cpu: 200m
+  memory: 200Mi
 
 node:
-  image: quay.io/karbon/ntnx-csi:v2.6.6
-  nodeSelector: {}
+  image: quay.io/karbon/ntnx-csi:v3.0.0-rc6
+  nodeSelector:
+    kubernetes.io/os: linux
   tolerations: []
+  cpu: 200m
+  memory: 200Mi
 
 sidecars:
   registrar:
-    image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.1
+    image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.8.0
   provisioner:
-    image: registry.k8s.io/sig-storage/csi-provisioner:v3.6.2
+    image: registry.k8s.io/sig-storage/csi-provisioner:v3.5.0
     imageLegacy: registry.k8s.io/sig-storage/csi-provisioner:v2.2.2
+  attacher:
+    image: k8s.gcr.io/sig-storage/csi-attacher:v3.4.0
   snapshotter:
-    image: registry.k8s.io/sig-storage/csi-snapshotter:v6.3.2
+    image: registry.k8s.io/sig-storage/csi-snapshotter:v6.2.1
     imageBeta: registry.k8s.io/sig-storage/csi-snapshotter:v3.0.3
+    # set it to true, if external-snapshotter sidecar should be deployed along with controller.
+    deploy: true
   resizer:
-    image: registry.k8s.io/sig-storage/csi-resizer:v1.9.2
+    image: registry.k8s.io/sig-storage/csi-resizer:v1.8.0
   livenessprobe:
-    image: registry.k8s.io/sig-storage/livenessprobe:v2.11.0
+    image: registry.k8s.io/sig-storage/livenessprobe:v2.10.0
+  healthmonitor:
+    image: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.9.0
+
+# PC versions on which Nutanix CSI Driver is supported currently. PLEASE DO NOT CHANGE.
+supportedPCVersions: "fraser-6.6-stable-pc-1,fraser-6.6-stable-pc-1.1,fraser-2023.3-stable-pc-0,fraser-2023.4-stable-pc-0"
```

Comment:
@subodh01 don't we need to remove fraser-6.6-stable-pc-1,fraser-6.6-stable-pc-1.1 
if minimum version is 2023.3 ??

Reply:
Yes.